src/test/resources/feature/cases/create_opportunity.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Create Opportunity
+
+  @DeleteOpportunity @CreateAccount @DeleteAccount @CreateCampaign
+  Scenario: create a new Opportunity with only required fields
```

Comment:
The scenario says only with required fields, but it seems that is with all fields.

Reply:
solved